### PR TITLE
Suppress warnings

### PR DESCRIPTION
**Pull request by joakim-hove**
Replaces: #844

### CODE REVIEW

**Comment by atgeirr** [comment]
A better way, thanks for the improvement.

**Comment by joakim-hove** [comment]
> A better way, thanks for the improvement.

I'll take that as a review - and merge!

File: opm/parser/eclipse/EclipseState/EclipseConfig.hpp [hunk 1]
```diff
@@ -38,7 +38,6 @@ namespace Opm {
     {
     public:
         EclipseConfig(const Deck& deck,
-                      const ParseContext& parseContext,
```
Comment (pgdr) [inline opm/parser/eclipse/EclipseState/EclipseConfig.hpp]: Removing this makes `class ParseContext` unnecessary.

File: opm/parser/eclipse/EclipseState/EclipseConfig.cpp [hunk 2]
```diff
@@ -34,7 +34,6 @@
 namespace Opm {
 
     EclipseConfig::EclipseConfig(const Deck& deck,
-                                 const ParseContext& parseContext,
```
Comment (pgdr) [inline opm/parser/eclipse/EclipseState/EclipseConfig.cpp]: No longer necessary to `#include <opm/parser/eclipse/Parser/ParseContext.hpp>`

**Pull request merged**